src/commands/Slots.js

```diff
@@ -0,0 +1,115 @@
+const { Command } = require('@dyno.gg/dyno-core');
+
+class Slots extends Command {
+    constructor(...args) {
+        super(...args);
+
+        this.aliases      = ['slots'];
+        this.module       = 'Fun';
+        this.description  = 'Spin the slots and win the jackpot! ...or loose everything.';
+        this.usage        = 'slots';
+        this.example      = 'slots';
+        this.cooldown     = 5000;
+        this.expectedArgs = 0;
+    }
+
+	execute({ msg }) {
+
+		const emojis = [":bullettrain_front:",":white_sun_small_cloud:",":lion_face:",":earth_africa:",":crown:",":bullettrain_front:",":white_sun_small_cloud:"] //Insert emoji strings of whatever you like
+
+		let frst = Math.floor(Math.random() * 5) + 1
+		let scnd = Math.floor(Math.random() * 5) + 1
+		let thrd = Math.floor(Math.random() * 5) + 1
+		let bonus = Math.floor(Math.random() * 10) + 6
+		let gameid = frst.toString() + scnd.toString() + thrd.toString()
+		let special = ["154", "215", "512", "451"]
+
+		let result = `\n<:dynov4:341008963254484992> ***DYNO SLOTS*** <:dynov4:341008963254484992>\n\n--------------------------\n\\\u2198     ${emojis[frst - 1]}  ${emojis[scnd - 1]}  ${emojis[thrd - 1]}     \\\u2199\n\n\\\u25B6    ${emojis[frst]}  ${emojis[scnd]}  ${emojis[thrd]}    \\\u25C0\n\n\\\u2197     ${emojis[frst + 1]}  ${emojis[scnd + 1]}  ${emojis[thrd + 1]}     \\\u2196\n--------------------------\n\n\n`

```

Comment:
Please break this string into multiple concatenations, like
```Javascript
let result = 'Dyno Slots\n';
result += 'more info here';
result += 'even more info here';
```
Or any other string concatenation or line-breaking method you like. But this huge string is hard to review and understand.